### PR TITLE
replace SQL with checkUserExisted(line 461)

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

File: server_api.py [hunk 1]
```diff
@@ -457,9 +457,9 @@ def register_no_right_user(data):
 
         db = mysql()
         db.connect()
-        sql = "SELECT count(*) FROM user WHERE user_name='" + str(data["bluetooth_id"]) + "'"
-        pure_result = db.query(sql)
-        if int(pure_result[0][0])<1:
+        with UserDao() as userDao:
+            pure_result = userDao.checkUserExisted(userName=str(data["bluetooth_id"]))
+        if int(pure_result)<1:
```
Comment (Billy4195) [inline server_api.py]: I think that above two lines can be more readable
* Change `pure_result` to `is_existed`
* `if int(pure_result) < 1:` -> `if not is_existed:`

Comment (stvreumi) [inline server_api.py]: Do you mean `if is_existed is None:`?

Comment (icecat2012) [inline server_api.py]: "if not is_existed:" will be correct.
"if is_existed is None:" when is_existed == 0 it will be false.

Comment (Billy4195) [inline server_api.py]: No, the `is_existed` will be int() in any case.
So the `if is_existed is None:` will cause fail.

**Comment by stvreumi** [comment]
OK, finish modification.

**Pull request merged**